build.gradle

```diff
@@ -72,6 +72,8 @@ subprojects { project ->
             check("StringSplitter", CheckSeverity.OFF)
             check("WildcardImport", CheckSeverity.ERROR)
             check("MissingBraces", CheckSeverity.ERROR)
+            check("TypeToString", CheckSeverity.ERROR)
+            check("SymbolToString", CheckSeverity.ERROR)
```

Comment:
Interesting, these are all suggestion-level, so even with `-Werror`, we don't get them by default...

Reply:
Yup.  I found these just browsing through the available checks one day a while back